src/main/java/techreborn/blocks/cable/CableBlock.java

```diff
@@ -136,6 +136,10 @@ private BlockState makeConnections(World world, BlockPos pos) {
 	private Boolean canConnectTo(IWorld world, BlockPos pos, Direction facing) {
```

Comment:
This method should really use the priamative type, not sure why i didnt.

I know you didnt add it in your PR but its an easy change to make a good time to do it.